src/jinja2/loaders.py

```diff
@@ -90,7 +90,8 @@ def get_source(self, environment, template):
             )
         raise TemplateNotFound(template)
 
-    def list_templates(self):
+    @staticmethod
+    def list_templates():
```

Comment:
Did you look at the code your PR changes? This is a method that gets overridden in some cases. So making it static in the base version with no functionality is somewhere between pointless and wrong, since a class overriding it will most likely *not* make it static...